FreeRTOS/FreeRTOS-Plus-TCP
# [IPv6] Reduce stack memory usage.

[IPv6] Reduce stack memory usage.

Description
-----------

- Output to address pointer in IPv6MC_GET_GROUP_ID instead of structure.
- Add FreeRTOS_in6addr_allnodes in static memory instread of stack.

Test Steps
-----------

Run protocol test cases IPv6/Datagram.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
